tests/spec/core/issues-notes-spec.js

```diff
@@ -197,16 +205,22 @@ describe("Core — Issues and Notes", function() {
     };
     const doc = await makeRSDoc(ops);
     const issueDiv1 = doc.getElementById("this-should-not-exist");
+
     expect(issueDiv1).toBeFalsy();
     const issueDiv2 = doc.getElementById("issue-1540");
+
     expect(issueDiv2).toBeTruthy();
     const issueDiv3 = doc.getElementById("i-should-be-here-too");
+
     expect(issueDiv3).toBeTruthy();
     const summarySection = doc.getElementById("issue-summary");
+
     expect(summarySection).toBeTruthy();
-    const { textContent } = summarySection.querySelector("[href='#issue-1540']");
-    expect(textContent).toBe("Issue 1540");
+    const { innerText } = summarySection.querySelector("[href='#issue-1540']");
+
+    expect(innerText).toBe("Issue 1540");
```

Comment:
textContent became innerText?

Reply:
odd... maybe came from the merged branch.